runtime/doc/vim_diff.txt

```diff
@@ -30,6 +30,7 @@ these differences.
 
 - 'backspace' defaults to "indent,eol,start"
 - 'encoding' defaults to "utf-8"
+- 'formatoptions'  includes "j" (see |fo-table|)
```

Comment:
Maybe just `formatoptions default to "tcqj"`, for consistency with `backspace` above.
I don't know about you, but I prefer the absolute definition over the relative one (which requires people to type `:help formatoptions`).